app/assets/stylesheets/_vocab_sheet.scss

```diff
@@ -196,19 +196,31 @@
 .vocab_sheet_drawing__size {
   &--2 {
     min-height: 290px;
```

Comment:
my scss is rudimentary - so 🐘 , but this repetition of 
```max height: number &img max-height: number - 10 ``` could be dry'd up with a mixin?

Reply:
oooh thats a great idea thanks!